schema/deploy/function_create_application_mutation_chain.sql

```diff
@@ -9,51 +9,16 @@ returns ggircs_portal.application
 as $function$
 declare
   new_id int;
-  form_result_id int;
   result ggircs_portal.application;
-  temp_row record;
-  form_result jsonb;
-  init_function varchar(1000);
-  query text;
 begin
 
   --Insert new value into application
-  insert into ggircs_portal.application(facility_id)
-  values (facility_id_input) returning id into new_id;
+  insert into ggircs_portal.application(facility_id, reporting_year)
+  values (facility_id_input, (select to_char(now(), 'YYYY')::int)) returning id into new_id;
```

Comment:
Add `-- TODO: use a current_reporting_year function (which should raise an exception if we are not in the reporting window`